model/AI/Vocabularies/EnergyConsumptionDescriptionUnitType

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# EnergyConsumptionDescriptionUnitType
+
+## Summary
+
+Specifies the unit of energy consumption
+
+## Description
+
+List different acceptable units of energy consumption. If the unit in which the energy consumption is recorded is not available here, please select other
+
+## Metadata
+
+- name: SafetyRiskAssessmentType
+
+## Entries
+
+- kW-H: Kilo Watt/ hour.
+- MJ: Mega Joule.
```

Comment:
```suggestion
- mJ: Mega Joule.
```

To be consistent with naming conventions

Reply:
Understand the consistency requirement, but that will made it read as "millijoule".

Reply:
Can we avoid the confusing by using the full name `megajoule` as an entry name ?